plugins/generic/externalFeed/classes/ExternalFeed.inc.php

```diff
@@ -20,6 +20,10 @@
 
 class ExternalFeed extends DataObject {
 
+	public function __construct() {
```

Comment:
Is this empty constructor necessary? (https://github.com/pkp/ojs/pull/1115#discussion_r161035708)